### PR TITLE
First Commit added a file by Soham

### PR DESCRIPTION
**Pull request by rushikeshghogare**

### CODE REVIEW

**Comment by rushikeshghogare** [comment]
this is correct code

**Review by SohamCool — CHANGES_REQUESTED** [review]
need very much improvement 
What is this? Even 1st standard kid will do a lot better than you. I 'll not merge this file. But don't be happy. I'll not gonna give you any kind of promotion to you.
Wait n watch

File: SohamFile.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class SohamFile{
+	public static void main(String[] args){
```
Comment (SohamCool) [inline SohamFile.java:2]: why we need main method?

Comment (SohamCool) [reply SohamFile.java:2]: We need to print a line thats why I use main method
Without it, it won't work at all

**Review by SohamCool — APPROVED** [review]
Ok then its ok

**Pull request merged**